worlds/wargroove2/Regions.py

```diff
@@ -0,0 +1,91 @@
+from BaseClasses import Region, Entrance
+from worlds.wargroove2.Levels import first_level, region_names, \
```

Comment:
Should be a relative import
```suggestion
from .Levels import first_level, region_names, \
```